Flow.Launcher.Test/FuzzyMatcherTest.cs

```diff
@@ -196,31 +201,38 @@ public void GivenQueryString_WhenAppliedPrecisionFiltering_ThenShouldReturnGreat
         [TestCase("sql serv man", MicrosoftSqlServerManagementStudio, SearchPrecisionScore.Regular, true)]
         [TestCase("sql studio", MicrosoftSqlServerManagementStudio, SearchPrecisionScore.Regular, true)]
         [TestCase("mic", MicrosoftSqlServerManagementStudio, SearchPrecisionScore.Regular, true)]
+        [TestCase("mssms", MicrosoftSqlServerManagementStudio, SearchPrecisionScore.Regular, true)]
+        [TestCase("msms", MicrosoftSqlServerManagementStudio, SearchPrecisionScore.Regular, true)]
```

Comment:
hmm ok, so acronym match doesnt have to match entire string

Reply:
Well, yes, for now. Because we include description in searching for now, which means that there will be quite a lot of unmatched acronym if we match the entire string.